src/exes/moc.ml

```diff
@@ -193,9 +193,9 @@ let argspec = [
   " Trap, don't throw an `Error`, when an IC call fails due to destination queue full or freezing threshold is crossed. Emulates behaviour of moc versions < 0.8.0.";
 
   (* persistence *)
-  "--enhanced-orthogonal-persistence",
```

Comment:
removing the old flag might just break code that specifies it. Maybe its best to keep the flag and add a new one to turn the feature off.